selfdrive/camerad/snapshot/snapshot.py

```diff
@@ -19,7 +19,7 @@ def snapshot():
   params = Params()
   front_camera_allowed = int(params.get("RecordFront"))
 
-  if params.get("IsTakingSnapshot") == b"1":
+  if params.get("IsOffroad") is not None or params.get("IsTakingSnapshot") == b"1":
```

Comment:
With a change to `  if params.get("IsOffroad") is None or params.get("IsTakingSnapshot") == b"1":
`, this works as expected

Reply:
oops, should be fixed now

Reply:
Tested fine. Thank you!